packages/security_center/lib/disk_encryption/disk_encryption_page.dart

```diff
@@ -33,109 +33,165 @@ class _Body extends ConsumerWidget {
-                    ),
-                  ],
+                ListTile(
+                  contentPadding: const EdgeInsets.symmetric(
+                    vertical: 8,
+                    horizontal: 16,
+                  ),
+                  leading: const Icon(YaruIcons.lock, size: 24),
+                  title: Text(
+                    l10n.recoveryKeyTPMEnabled,
+                    style: Theme.of(context).textTheme.titleSmall,
+                  ),
                 ),
-              AuthMode.passphrase => Column(
-                  crossAxisAlignment: CrossAxisAlignment.start,
-                  children: [
-                    Text(
-                      l10n.recoveryKeyPassphraseHeader,
-                      style: Theme.of(context).textTheme.titleMedium,
+                if (data != AuthMode.none) ...[
+                  const Divider(),
+                  ListTile(
+                    contentPadding: const EdgeInsets.symmetric(
+                      vertical: 8,
+                      horizontal: 16,
                     ),
-                    const SizedBox(height: 8),
-                    Text(l10n.recoveryKeyPassphraseBody),
-                    const SizedBox(height: 16),
-                    OutlinedButton(
-                      onPressed: () {
-                        ref
-                            .read(changeAuthDialogModelProvider.notifier)
-                            .authMode = AuthMode.passphrase;
-                        showChangeAuthDialog(context, AuthMode.passphrase);
-                      },
-                      child: Text(l10n.recoveryKeyPassphraseButton),
+                    leading: const Icon(YaruIcons.ok_simple, size: 24),
+                    title: Text(
+                      data == AuthMode.pin
+                          ? l10n.recoveryKeyPinEnabled
+                          : l10n.recoveryKeyPassphraseEnabled,
+                      style: Theme.of(context).textTheme.titleSmall,
                     ),
-                  ],
-                ),
-              AuthMode.none => const SizedBox.shrink(),
-            },
-          ],
-        );
-      },
-      error: (e, stack) => YaruInfoBox(
-        title: Text(l10n.diskEncryptionPageError),
-        subtitle: Text(e.toString()),
-        yaruInfoType: YaruInfoType.danger,
+                  ),
+                ],
+              ],
+            ),
+          ),
+          const SizedBox(height: 16),
+          Text(
+            l10n.recoveryKeyTPMExplanationBody,
+            textAlign: TextAlign.left,
+          ),
+          const SizedBox(height: 16),
+          Text(
+            l10n.diskEncryptionPageRecoveryKey,
+            style: Theme.of(context).textTheme.titleMedium,
+            textAlign: TextAlign.left,
+          ),
+          const SizedBox(height: 16),
+          MarkdownText(
+            '${l10n.diskEncryptionPageStoreYourKey} ${l10n.diskEncryptionPageLearnMore.link(_learnMoreUrl)}',
```

Comment:
I think concatenating l10n strings like this will potentially case issues with RTL languages.